src/views/EditorHome.vue

```diff
@@ -570,6 +571,24 @@ onMounted(async () => {
     });
   }
 
+  // ショートカットキーの除外設定
+  // 除外するなら`true`を返す
```

Comment:
細かいですが、ショートカットキーを除外するかどうかと勘違いしちゃうかも･･･？
こうとかですかねぇ。。
```suggestion
  // ショートカットキーを止める条件の設定
  // 止めるなら`true`を返す
```